metals/src/main/scala/scala/meta/internal/metals/codeactions/CreateNewSymbol.scala

```diff
@@ -37,15 +37,40 @@ class CreateNewSymbol() extends CodeAction {
       )
     }
 
+    def createNewMethod(
```

Comment:
You need to add:
```
override type CommandData = l.TextDocumentPositionParams
def command: Option[ActionCommand] = Some(ServerCommands.InsertInferredMethod)
```
somewhere above in the class

Reply:
done in https://github.com/scalameta/metals/pull/6877/commits/12ec7f86a0245b74b3a1b5597d466250bcac58ec

That helped, but the real issue was that the `handleCommand` wasn't defined and inferMethod needs it to apply edits.

Reply:
Ach, yes I hoped it would get suggested by the compiler automatically. At least that was the idea when I added those fields in CodeAction.

Good catch!